doc/lms.pgsql

```diff
@@ -60,6 +56,25 @@ CREATE TABLE pds (
     CONSTRAINT pds_supplierid_ukey UNIQUE (fullnumber, supplierid)
 );
 
+/* --------------------------------------------------------
+  Structure of table "pdcontents"
+-------------------------------------------------------- */
+DROP TABLE IF EXISTS pdcontents CASCADE;
+DROP SEQUENCE IF EXISTS pdcontents_id_seq;
+CREATE SEQUENCE pdcontents_id_seq;
+CREATE TABLE pdcontents (
+    id integer DEFAULT nextval('pdcontents_id_seq'::text) NOT NULL,
+    pdid integer NOT NULL
+        CONSTRAINT pdcontents_pdid_fkey REFERENCES pds (id) ON DELETE CASCADE ON UPDATE CASCADE,
+    netvalue numeric(9,2) NOT NULL,
```

Comment:
To jest dla każdej pozycji:
Czy nie lepiej byłoby: currency, currencyvalue, taxid, plntaxvalue?